### PR TITLE
Update manage-locations.js

### PR DESCRIPTION
**Pull request by Quintis1212**
Hi , I added label to manage location menu on "Navigation screen: regression of improved menu locations select placeholder #30330" issue from new branch

### CODE REVIEW

**Comment by kevin940726** [comment]
There are still come conflicts, could you rebase the branch onto `trunk` :)?

**Comment by talldan** [comment]
I think the same thing happened as with the previous PR.

There's some info here about how to sync the version of trunk on your fork with the one in this repo:
https://github.com/WordPress/gutenberg/blob/trunk/docs/contributors/code/git-workflow.md

**Comment by getdave** [comment]
Hi @Quintis1212. How are things going? Do you think you will be able to sync your fork with the Gutenberg repo? Anything I can help with?

**Comment by Quintis1212** [comment]
> Hi @Quintis1212. How are things going? Do you think you will be able to sync your fork with the Gutenberg repo? Anything I can help with?

Hi @getdave  ) I don't know what I must to do , my other branches are working fine

**Comment by getdave** [comment]
Hi @Quintis1212. Ok no worries. Your fork of Gutenberg is out of sync with the Core Gutenberg repo `trunk`. That's why we see `packages/edit-navigation/src/components/header/manage-locations.js` mentioned under `Conflicting files` on this PR page.

You need to pull in the changes from the `trunk` branch of the Core Gutenberg repo. To do this:

1. Add the _Core_ repo as the alias `upstream` - `git remote add upstream https://github.com/WordPress/gutenberg`.
2. Pull the latest `trunk` from `upstream` - `git fetch upstream trunk`.
3. Switch to your branch on your fork - `git checkout 123-L`.
4. Rebase your branch with changes from upstream - `git rebase upstream/trunk`.
5. Resolve conflicts.
6. Once done then push back to your branch `git push -u origin 123-L` (this assumes your fork is called `origin`). 

Note for step 6, if you get any errors saying your local branch may overwrite the remote you can try appending `--force-with-lease` to the end of the command (eg: `git push origin 123-L --force-with-lease`).

I hope that helps? Let me know how you get on. I'd love to help you get this landed.

**Comment by talldan** [comment]
@Quintis1212 I'd also recommend updating your fork's `trunk` branch up-to-date with Gutenberg's `trunk` branch. It's good to do this before creating a new topic branch that you plan to work on.

Github now has a handy feature to help with that. You can click this 'Fetch and merge' button for your fork:
Screenshot 2021-05-11 at 12 43 27 pm

And then the following will bring those changes to your local machine:
1. `git checkout trunk`
2. `git pull`

**Comment by getdave** [comment]
@Quintis1212 It looks like the file you are editing has moved to here:

https://github.com/WordPress/gutenberg/blob/305cf6ff09a709a44037f044a6c12994a593af65/packages/edit-navigation/src/components/inspector-additions/manage-locations.js#L100-L105

You might find it easier to simply create a new PR based off of `trunk` and then make your changes again in the new file (`packages/edit-navigation/src/components/inspector-additions/manage-locations.js`).

I'll be happy to review and approve as soon as you get to it.

Let me know if I can help in any way. Thanks again.

**Comment by Quintis1212** [comment]
Hello  @getdave  ,@talldan  ) thank you for help ) I deleted my local gutenberg plugin and cloned  gutenberg from my fork to open new PR without conflicts , but I have an error :
`npm ERR! code EBADENGINE
npm ERR! engine Unsupported engine
npm ERR! engine Not compatible with your version of node/npm: gutenberg@10.1.1
npm ERR! notsup Not compatible with your version of node/npm: gutenberg@10.1.1
npm ERR! notsup Required: {"node":">=10.0.0","npm":">=6.9.0 <7"}
npm ERR! notsup Actual:   {"npm":"7.10.0","node":"v14.16.0"}
`
I must have the lower version of npm to run npm ci  ?

**Comment by getdave** [comment]
> Hello @getdave ,@talldan ) thank you for help ) I deleted my local gutenberg plugin and cloned gutenberg from my fork to open new PR without conflicts , but I have an error :
> `npm ERR! code EBADENGINE npm ERR! engine Unsupported engine npm ERR! engine Not compatible with your version of node/npm: gutenberg@10.1.1 npm ERR! notsup Not compatible with your version of node/npm: gutenberg@10.1.1 npm ERR! notsup Required: {"node":">=10.0.0","npm":">=6.9.0 <7"} npm ERR! notsup Actual: {"npm":"7.10.0","node":"v14.16.0"} `
> I must have the lower version of npm to run npm ci ?

At the moment Gutenberg requires `npm` less than `7`.

I had the same issue so I temporarily downgraded `npm` to `6.9.0`.

```
npm install -g npm@6.9.0
```

**Comment by Quintis1212** [comment]
Thank you @getdave) Do you know who is working for allowing 7 and high npm ? I want to help

**Comment by getdave** [comment]
> Thank you @getdave) Do you know who is working for allowing 7 and high npm ? I want to help

I don't specifically but if you ask in [the `core-js` channel in WordPress.org Slack](https://wordpress.slack.com/archives/C5UNMSU4R) someone will probably be able to help.

File: packages/edit-navigation/src/components/header/manage-locations.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ export default function ManageLocations() {
 			labelPosition="top"
 			value={ menuLocation.menu }
 			options={ [
-				{ value: 0, label: __( '-' ) },
+				{ value: 0, label: __( '— Select a Menu —' ) },
```
Comment (getdave) [inline packages/edit-navigation/src/components/header/manage-locations.js:37]:
```suggestion
				{ value: 0, label: __( 'Select a Menu' ) },
```

Looking at [the i18n guidelines](https://codex.wordpress.org/I18n_for_WordPress_Developers#Best_Practices) I'm not sure we should include the characters inside the translated string.

**Review by getdave — COMMENTED** [review]
Happy to approve this if we can update to remove the characters from inside the string to be translated.

**Comment by getdave** [comment]
Hey @Quintis1212. Looks like this is nearly ready to land although there is now another conflict with the `packages/edit-navigation/src/components/header/manage-locations.js` file (same process as before). Happy to approve if we can address the final point in my code review.

I'd love to help you land this PR. Let me know if/how I can help.

**Comment by getdave** [comment]
@Quintis1212 Is there anything I can do to help you further here?

**Comment by Quintis1212** [comment]
Hi @getdave , I created new branch and new PR - https://github.com/WordPress/gutenberg/pull/32423  , but it seems there is a conflict

**Pull request closed without merging**